### PR TITLE
Starting first setup

### PR DESCRIPTION
**Pull request by gbrancaglione**
`npm start`

image

Fiz esse tutorial aqui:

Here's a step-by-step guide to help you start a basic Node.js project using Express.js, connected to a PostgreSQL database, and using TypeScript:

### Day 1: Setting Up the Project

1. **Install Node.js and npm**:
   - Make sure you have Node.js and npm installed. You can download it from [here](https://nodejs.org/).

2. **Initialize a New Project**:
   - Create a new directory for your project and navigate into it:
     ```bash
     mkdir my-express-api
     cd my-express-api
     ```
   - Initialize a new Node.js project:
     ```bash
     npm init -y
     ```

3. **Install Dependencies**:
   - Install Express.js:
     ```bash
     npm install express
     ```
   - Install TypeScript and necessary TypeScript types:
     ```bash
     npm install typescript ts-node @types/node @types/express --save-dev
     ```

4. **Set Up TypeScript**:
   - Initialize a TypeScript configuration file:
     ```bash
     npx tsc --init
     ```
   - Edit the `tsconfig.json` file to your preferences. Ensure you have the following settings:
     ```json
     {
       "compilerOptions": {
         "target": "ES6",
         "module": "commonjs",
         "outDir": "./dist",
         "rootDir": "./src",
         "strict": true,
         "esModuleInterop": true
       }
     }
     ```

5. **Create Project Structure**:
   - Create the necessary folders and files:
     ```bash
     mkdir src
     touch src/index.ts
     ```

### Day 2: Basic Express Setup

1. **Write a Basic Server**:
   - In `src/index.ts`, set up a basic Express server:
     ```typescript
     import express from 'express';

     const app = express();
     const port = process.env.PORT || 3000;

     app.get('/', (req, res) => {
       res.send('Hello World!');
     });

     app.listen(port, () => {
       console.log(`Server is running on port ${port}`);
     });
     ```

2. **Add Start Script**:
   - Update `package.json` to include a start script:
     ```json
     "scripts": {
       "start": "ts-node src/index.ts"
     }
     ```

3. **Run Your Server**:
   - Start your server:
     ```bash
     npm start
     ```

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+
+const app = express();
+const port = process.env.PORT || 3000;
```
Comment (gbrancaglione) [inline src/index.ts:4]: Vou ter que instalar a lib de `dotenv` para isso virar uma env var.

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
+{
```
Comment (gbrancaglione) [inline tsconfig.json:1]: Não perguntei, mas minha sugestão aqui é usar o `Typescript`.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "ai-calendar",
+  "version": "1.0.0",
```
Comment (gu-maia) [inline package.json]: Não é melhor deixar a versão 1.0.0 pra primeira release redondinha?

**Review by gu-maia — APPROVED** [review]
Só consideraria usar semantic versioning no pacote

Tirando isso, LGTM. Vou tentar rodar aqui!

**Pull request merged**